src/core/runtime/dns-nodejs.js

```diff
@@ -1,13 +1,14 @@
 'use strict'
 
 const dns = require('dns')
-const _ = require('lodash')
+const flatten = require('lodash.flatten')
```

Comment:
can we use https://github.com/angus-c/just#just-flatten-it here ?

Reply:
I think we need to sweep the entire codebase to remove lodash, then we can choose blessed replacements for it's various functions.